src/components/Table/index.tsx

```diff
@@ -101,8 +102,8 @@ class GnoTable extends React.Component<any, any> {
     }))
   }
 
-  getEmptyStyle = (emptyRows) => ({
-    height: FIXED_HEIGHT * emptyRows,
+  getEmptyStyle = (emptyRows, isEmpty) => ({
+    height: isEmpty ? `calc(100vh - ${FIXED_EMPTY_HEIGHT}px)` : FIXED_HEIGHT * emptyRows,
```

Comment:
`getEmptyStyle` is always called when `isEmpty` is true, so you don't need to pass any arguments 🙂
FIXED_HEIGHT can be removed, too.